server/server.go

```diff
@@ -141,9 +176,27 @@ func (server *HTTPServer) readReportForCluster(writer http.ResponseWriter, reque
 	} else if err != nil {
 		log.Println("Unable to read report for cluster", err)
 		responses.SendInternalServerError(writer, err.Error())
+	}
+
+	rulesContent, rulesCount, err := server.getContentForRules(writer, report)
+	if err != nil {
+		return
```

Comment:
ah I see, the response is handled in previous method. It is a bit unusual, prob. worth to move the `Send*` here?

Reply:
I was considering it, but I wanted to use the same "style" of responses like it's used here (in this function too), 20L before is
```
	organizationID, err := readOrganizationID(writer, request)
	if err != nil {
		// everything has been handled already
		return
	}

	clusterName, err := readClusterName(writer, request)
	if err != nil {
		// everything has been handled already
		return
	}
```
I added the comment for completeness

Reply:
ok, TY